File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/datatypes.java
import java.util.Scanner;

public class question2 {
    public static int dataTypes(String type) {
        // Write your code here
        type = type.toLowerCase();
        switch (type) {
            case "integer":
                return 4;
            case "long":
                return 8;
            case "float":
                return 4;
            case "double":
                return 8;
            case "character":
                return 1;
            default:
                return (-1);

        }
    }

    public static void main(String[] args) {
        try (Scanner read = new Scanner(System.in)) {
            String dataty = read.next();
            System.out.println(dataTypes(dataty));
        }
    }
}

File: SecondYear/chikkibum_BhaskarPandey_2226cseai1118_2/Week_1/datatypes.cpp
int dataTypes(string type) {
	if(type=="Integer"){
		return 4;
	}
	else if(type=="Float"){
		return 4;
	}
	else if(type=="Long"){
		return 8;
	}
	else if(type=="Double"){
		return 8;
	}
	else if(type=="Character"){
		return 1;
	}
	return -1;
}